lib/datadog/tracing/contrib/action_pack/action_dispatch/instrumentation.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require_relative '../../../metadata/ext'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module ActionPack
+        module ActionDispatch
+          # Instrumentation for ActionDispatch components
+          module Instrumentation
+            module_function
+
+            def set_http_route_tag(http_route)
+              return if http_route.nil?
+
+              active_span = Tracing.active_span
+              return unless active_span
+
+              active_span.set_tag(
+                Tracing::Metadata::Ext::HTTP::TAG_ROUTE,
+                http_route.gsub(/\(.:format\)$/, '')
```

Comment:
In Ruby the end of string is supposed to be `\z`, `$` means end of line.

Reply:
fixed